ogarniacz_mp3.py

```diff
@@ -113,7 +113,19 @@ def downloaded_to_processing(source_dir_path, target_dir_path): #ścieżka do ka
                 except BadZipfile:
                     print(f"Plik {f} nie jest plikiem .zip.")
 
-                    
+def getting_objects(processing_dir_path):
+    dir_objects = []
+    file_objects = []
+    for dir_path, sub_dirs, files in os.walk(processing_dir_path, topdown=False):
```

Comment:
`topdown` nie jest tutaj potrzebny - nie przenosimy/usuwamy plików ani katalogów, więc to bez znaczenia w jakiej kolejności będziemy iterować po tych katalogach.